modules/templates/helper.go

```diff
@@ -188,6 +188,7 @@ func NewFuncMap() template.FuncMap {
 
 		"FilenameIsImage": FilenameIsImage,
 		"TabSizeClass":    TabSizeClass,
+		"ReviewTypeColor": ReviewTypeColor,
```

Comment:
To avoid bloating the template helpers, I think it could be done like this:

```
func (reivew issue_model.Review) HTMLColorName() string {
    ....
    return "green" 
}
```


Then in template:

```
{{ReviewTypeColor .HTMLColorName}}
```

Reply:
> {{.Review.HTMLColorName}}

Interesting. It seems strange calling a function without the `()`, but I guess this is a oddity of go templates?

Reply:
I have done one similar refactoring PR: #24570


If you want to pass arguments: 

`{{.Review.HTMLColorName arg1 arg2}}`

Reply:
Thanks, I will try to replicate that here.